Apps/Motioneye/docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+name: motioneye
+services:
+  app:
+    cpu_shares: 10
+    command: []
+    container_name: motioneye
+    deploy:
+      resources:
+        limits:
+          memory: 1794M
+    environment:
+      - TZ=Europe/Berlin
+    image: ccrisan/motioneye:master-amd64
+    labels:
+      icon: https://cdn.jsdelivr.net/gh/IceWhaleTech/CasaOS-AppStore@main/Apps/Motioneye/icon.png
+    ports:
+      - target: 8765
+        published: "8765"
+        protocol: tcp
+    restart: unless-stopped
+    volumes:
+      - type: bind
+        source: /DATA/AppData/motioneye/config
+        target: /etc/motioneye
+      - type: bind
+        source: /DATA/Media/motioneye
+        target: /var/lib/motioneye
+    x-casaos:
+      ports:
+        - container: "8765"
+          description:
+            en_us: Motioneye Web Port
+          host: "8765"
+      volumes:
+        - container: /etc/motioneye
+          description:
+            en_us: Config files
+        - container: /var/lib/motioneye
+          description:
+            en_us: Media files
+    devices: []
+    cap_add: []
```

Comment:
Please remove properties with empty list. There's a known issue that this will break the AppStore.